terminus-ui/drawer/testing/src/test-components.ts

```diff
@@ -0,0 +1,248 @@
+import {
+  Component, ElementRef, NgModule, ViewChild,
+} from '@angular/core';
+
+import {
+  TsDrawerComponent,
+  TsDrawerModes, TsDrawerModule,
+  TsDrawerPosition,
+  TsDrawerContainerComponent, TsDrawerContentComponent,
+} from '@terminus/ui/drawer';
```

Comment:
a) Multiple lines.
b) It doesn't look like `TsDrawerContainerComponent` is being used
c) Imports not ordered